README.md

```diff
@@ -346,7 +346,7 @@ M.as_completed = function(tasks) end
 
 [The `coop.lsp.client` module](https://github.com/gregorias/coop.nvim/blob/main/lua/coop/lsp/client.lua)
 provides a task function implementation for
-[`Client:request`](https://neovim.io/doc/user/lsp.html#Client%3Arequest()).
+[`Client:request`](<https://neovim.io/doc/user/lsp.html#Client%3Arequest()>).
```

Comment:
Why are we adding angle brackets? If this is by accident, could you remove it from the PR?